about.html

```diff
@@ -17,6 +17,38 @@
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/svg.js/2.6.3/svg.js"></script>
 	<script src="assets/scripts/scripts.js"></script>
 	<script src="https://use.fontawesome.com/d11bbf05c5.js"></script>
+
```

Comment:
all of the content inside this script tag should go in a separate file, which is linked at the top of the file (like line 18)